uportal-war/src/main/java/org/apereo/portal/portlet/container/services/PortletCookieServiceImpl.java

```diff
@@ -332,6 +332,7 @@ protected Cookie convertToCookie(IPortalCookie portalCookie, boolean secure) {
     	cookie.setComment(this.comment);
     	cookie.setMaxAge(this.maxAge);
     	cookie.setSecure(secure);
+        cookie.setHttpOnly(true);
```

Comment:
It should match the indentation.  Our current standard (iirc) is 4 spaces, no tabs.